src/main/java/website/frontrow/board/Player.java

```diff
@@ -149,7 +168,8 @@ public void setLives(int l)
     public void loseLife()
     {
     	lives--;
-    	addToLog("[PLAYER]\t Player lost a life, total lives is now: " + lives);
+        observers.forEach(o -> o.livesChanged(this));
```

Comment:
Indenting, line 170 has a tab, line 171 only has spaces.
There's a visible difference, because of this.